HACKING.md

```diff
@@ -83,7 +84,7 @@ e.g.; to update the list for 16.04,
 
 Install the needed dependencies.
 
-    sudo apt install build-essential python3-dev libapt-pkg-dev libsodium-dev gcc libffi-dev
+    sudo apt install libarchive13 python3-pip build-essential python3-dev libapt-pkg-dev libsodium-dev gcc libffi-dev squashfs-tools
```

Comment:
what about using sudo apt build-dep -y ./ so we don't have to update it every time?
The downside is that it requires the src apt repos.